.nais/naiserator-dev.yaml

```diff
@@ -41,35 +41,46 @@ spec:
       claims:
         extra:
           - "NAVident"
+  accessPolicy:
+    outbound:
+      external:
+        - host: "login.microsoftonline.com"
+        - host: "internarbeidsflatedecorator-q1.dev-fss-pub.nais.io"
+        - host: "modiacontextholder-q1.dev-fss-pub.nais.io"
+        - host: "ereg-services-q1.dev-fss-pub.nais.io"
+      rules:
+        - application: isdialogmote
+        - application: syfoveileder
+        - application: syfoperson
+        - application: syfomoteoversikt-redis
   envFrom:
     - secret: syfomoteoversikt-session-key
-  webproxy: true
   ingresses:
-    - "https://syfomoteoversikt.dev.intern.nav.no"
+    - "https://syfomoteoversikt.intern.dev.nav.no"
   env:
     - name: NODE_ENV
       value: "production"
     - name: AUTH_REDIRECT_URI
-      value: "https://syfomoteoversikt.dev.intern.nav.no/oauth2/callback"
+      value: "https://syfomoteoversikt.intern.dev.nav.no/oauth2/callback"
     - name: REDIS_HOST
       value: "syfomoteoversikt-redis"
     - name: EREG_HOST
-      value: "ereg-services-q1.dev.intern.nav.no"
+      value: "https://ereg-services-q1.dev-fss-pub.nais.io"
     - name: INTERNARBEIDSFLATEDECORATOR_HOST
-      value: "internarbeidsflatedecorator-q1.dev.intern.nav.no"
+      value: "https://internarbeidsflatedecorator-q1.dev-fss-pub.nais.io"
     - name: ISDIALOGMOTE_HOST
-      value: "isdialogmote.intern.dev.nav.no"
+      value: "http://isdialogmote"
```

Comment:
Hvorfor er noen `http` og noen `https` egentlig?

Reply:
De som bruker service discovery i stedet for full url bruker bare http 👍

Reply:
Vi snakka litt om det her fysisk, og det meste står oppsummert her: https://docs.nais.io/clusters/service-discovery/. Alle servicer på gcp følger navneformatet: http://<service-name>.<namespace>.svc.cluster.local, men de har altså en kortversjon hvis de er i samme namespace: http://<another-service>